package.json

```diff
@@ -33,7 +35,7 @@
     "@types/express": "4.17.1",
     "@types/morgan": "1.9.3",
     "eslint": "8.0.1",
-    "ts-node": "^10.4.0",
-    "typescript": "^4.5.2"
+    "ts-node": "10.4.0",
+    "typescript": "4.5.2"
```

Comment:
I'm sure these can be devDependencies. The compiled javascript won't use these packages